Hospital-System/HospitalSystem.Common/Models/DTOs/Appointment/AppointmentDTO.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace HospitalSystem.Common.Models.DTOs.Appointment;
+
+public class AppointmentDTO
+{
+    public string Diagnosis { get; set; } = string.Empty;
+    public string AdditionalInformation { get; set; } = string.Empty;
+    public Guid DoctorTimeSlotId { get; set; }
```

Comment:
TimeSlotId (without doctor's is better). Cause timeslot is not something specific to doctor, it also affects patient